AudioKitSynthOne/Keyboard/KeyboardSettingsViewController.swift

```diff
@@ -75,3 +77,36 @@ class KeyboardSettingsViewController: UIViewController {
 
     }
 }
+
+/**
+Accessibility Functions
+*/
+extension KeyboardSettingsViewController {
+	
+	
+	private func setupAccessibility() {
+		setOctaveRangeSegmentAccessibility()
+		setLabelModeSegment()
+		setKeyboardModeSegment()
+	}
+	
+	
+	private func setOctaveRangeSegmentAccessibility() {
+		octaveRangeSegment.subviews[0].accessibilityLabel = "1"
+		octaveRangeSegment.subviews[1].accessibilityLabel = "2"
+		octaveRangeSegment.subviews[1].accessibilityLabel = "3"
+	}
+	
+	private func setLabelModeSegment() {
+		labelModeSegment.subviews[0].accessibilityLabel = "None"
+		labelModeSegment.subviews[1].accessibilityLabel = "C"
+		labelModeSegment.subviews[2].accessibilityLabel = "All"
+		
+	}
+	
+	private func setKeyboardModeSegment() {
+		keyboardModeSegment.subviews[0].accessibilityLabel = "White"
+		keyboardModeSegment.subviews[1].accessibilityLabel = "Black"
```

Comment:
I would suggest using `NSLocalizedString` on all the localizable strings here.

Reply:
OH Good Catch I totally Forgot about those.